modules/search/instant-search/components/search-result-product.jsx

```diff
@@ -34,7 +35,7 @@ class SearchResultProduct extends Component {
 					/>
 				</h3>
 				{ firstImage && (
-					<img
+					<PhotonImage
```

Comment:
Can we more dynamically select the size here, or slightly increase the default size (300x300?). Just looks like this has shrunk the results a lot so they are no longer using the full width:

![Screen Shot 2019-11-08 at 7 31 31 AM](https://user-images.githubusercontent.com/820871/68484096-e80ef400-01f9-11ea-9d73-d49b6bbd158c.png)

It seems like even on mobile devices 160x160 is going to be too small.

Reply:
👍I'll bump to 300x300. I suspect we'll have to do some more tuning when we have the final layout sorted.